src/plugins/roomslist/view.js

```diff
@@ -55,7 +55,7 @@ export class RoomsList extends CustomElement {
     async openRoom (ev) {
         ev.preventDefault();
         const target = u.ancestor(/** @type {HTMLElement} */(ev.target), '.open-room');
-        const name = target.textContent;
+        const name = target.querySelector(".room-name")?.textContent;
```

Comment:
Thank you for the fix.

I think a slightly better solution is to put the name as a `data-` attribute on the `a` element, similar to `data-room-jid`.

Then this becomes: `const name = target.getAttribute('data-room-name');`